app/src/utils/nfcScanner.ts

```diff
@@ -52,8 +52,6 @@ export const parseScanResponse = (response: unknown) => {
 };
 
 export const scan = async (inputs: Inputs) => {
-  await configureNfcAnalytics();
```

Comment:
this [is now called](https://github.com/selfxyz/self/pull/1187/files#diff-9fa027bd6430742b3931e0e2e7f0c77eedbc73bf0a7a48fc6466f5412532b1edR327) before calling `scan` as preparation to move the code over to the SDK